Core/include/Acts/TrackFitting/detail/KLMixtureReduction.hpp

```diff
@@ -129,10 +129,6 @@ class SymmetricKLDistanceMatrix {
     setAssociated(n, m_mask, [&](std::size_t, std::size_t) { return false; });
   }
 
-  void setAssociatedDistances(std::size_t n, ActsScalar value) {
-    setAssociated(n, m_mask, [&](std::size_t, std::size_t) { return value; });
-  }
-
```

Comment:
This change is unrelated right?

Reply:
No, this actually triggers the `-Wfloat-conversion`. I actually introduces this function/buga few weeks ago during a refactoring, but it was unused so the bug was never visible in the results...